### ISSUE TITLE
Help needed

### ISSUE THREAD

**Issue by acemodder** [arkayenro/arkinventory#783]
Can somebody help me understand how to create a rule /catergory for item grouping.

For example say I wanted to have a custom group with the garrison heartstone, the hearthstone and admiral's compass into a seperate Bar - how would I do this? 

I'm totally confused.

**Comment by Yanngo** [comment]
1 .Go to Settings >> Categories >> Custom and make a category with the name you want. Press OK on everything to close dialogues if there are any.
![make a custom category](https://cloud.githubusercontent.com/assets/17704069/17040384/064da626-4fa8-11e6-8945-1415ba2d9918.jpg)

2. Go to edit mode. There is a button at the top of ArkInventory in bag mode.  Choose a bar. They have numbers. And assign the category you've created to that bar by clicking on the name of the category.
![assign category to bar](https://cloud.githubusercontent.com/assets/17704069/17040406/2061e40a-4fa8-11e6-867a-e691164b2204.jpg)

3. In edit mode assign the item to the category.
![assign item to category](https://cloud.githubusercontent.com/assets/17704069/17040421/2f4c4e2e-4fa8-11e6-84ec-dfe97f2b5a4a.jpg)

**Comment by acemodder** [comment]
Thank you working just how I want it now.

**Comment by acemodder** [comment]
Hi.

I've noticed that very often items are not going into the correct category.  

See below showing that bar 4 has "follower crap" group assigned to it that I created.
![screenshot 2016-07-23 22 01 46](https://cloud.githubusercontent.com/assets/20584654/17080282/3a4cb64a-5121-11e6-9290-84356dc28652.png)
I have the blackrock weaponry assigned to "follower crap" but 75% of the time when I open my bag it doesnt assign it into bar 4.  See below.
![screenshot 2016-07-23 22 00 59](https://cloud.githubusercontent.com/assets/20584654/17080291/80d7d608-5121-11e6-9e47-6092f3a4692b.png)

If I keep closing and opening my bags, eventually it shows in the correct bar 4.    Any idea's why this is?
